mjrt/Cargo.toml

```diff
@@ -1,14 +1,13 @@
 [package]
-name = "mjrt-bin"
+name = "mjrt"
 version = "0.1.0"
 authors = ["Christian Schwarz <me@cschwarz.com>"]
 edition = "2018"
-links = "mjrt_src"
+links = "mjrt"
```

Comment:
shouldn't this be `mjrt_impl`?

Reply:
I think we could just remove this line, we don't rely on the `DEP_MJRT_.*` environment variable anymore.